setup-xcode-signing.md

```diff
@@ -4,20 +4,25 @@ In order to build an iOS app with custom entitlements, you'll need to setup _cod
 
 > 💡 Certificates are saved to your local Keychain so you don't need to set them up often.
 
-![code-signing-page](https://user-images.githubusercontent.com/9664363/112059266-db521e00-8b18-11eb-9538-2a31c6bd6e39.png)
-
-
 1. Open your iOS project in Xcode by running `xed ios` or `open ios/<yourproject>.xcworkspace` in your Terminal. If your Expo project doesn't have an `ios` folder, run `npx expo prebuild -p ios` to generate one locally.
 2. Select the first project in the **navigator** then the target with the matching native in the "project and targets list".
 3. Select "Signing & Capabilities", ensure you have "Automatically manage signing" selected.
 4. Ensure a "Development Team" is selected, this may require that you sign-in to your Apple Developer account.
    1. Press the "Add Account" button next to the team label. Follow the sign in flow.
-5. Select a physical device to build the app onto, Xcode will add this device to your account.
-6. Build the app onto your device.
-   1. You may need to "trust" the device you just added to the Xcode "Development Certificate", this is done in your iOS device.
-   2. Navigate: Settings > General > Device Management > [your new certificate] > Trust.
+
+   ![](./assets/setup-xcode-signing/01-add-signing-account.png)
+
+5. Select a physical device from the dropdown at the top-center of the UI. Xcode will add this device to your account.
+6. Build the app onto your device by pressing the "Play" symbol at the top-left of the the UI.
```

Comment:
These were the two steps that confused me before. I added a screenshot for clarity.